_posts/2020-03-11-sr2020-covid-19-updates.md

```diff
@@ -0,0 +1,40 @@
+---
+layout: news
+title: COVID-19 Coronavirus Student Robotics Updates
+---
+
+As you will be aware, the COVID-19 situation is quickly evolving, having recently been delcared a pandemic. 
+Many large events are being cancelled and we want to be public about how we are responding to the latest news.
+
+The Competition Team are working closely with the Student Robotics trustees so that we can make the most sensible decision.
+
+## The Competition
+
+At the time of writing, we are carefully considering whether the competition can go ahead. We will be announcing our decision by Wednesday 18th March at the very latest. 
+Considering this is an event for minors, we will be taking decisions with an abundance of caution. 
+
+We are in contact with Reading University Students' Union who are continually reviewing the situation, and mitigating the risks associated with Coronavirus.
+
+The Competition Team will be meeting with the trustees at the start of next week (16th March) to decide whether the competition can go ahead.
+
+### In the case of the competition going ahead
+
+We will be briefing all volunteers and attendees about best hygiene practices, both in advance and on the day, ensuring they're followed throughout the competition. 
+
+We expect all teams who are attending the competition to follow the [guidelines][phe-guidelines] on COVID-19 from Public Health England.
+
+### In the case of the competition being postponed
+
+We are considering weekends in late June / early July as alternatives for the competition. 
+If the competition is postponed, we will be surveying team leaders to ensure we select the most appropriate date.
```

Comment:
It's implied by the section title that this'll be done if the event is postponed.

```suggestion
We will be surveying team leaders to ensure we select the most appropriate date.
```